README.md

```diff
@@ -1,9 +1,73 @@
-# autofs #
+# puppet-module-autofs
+
+Manage autofs
+
 ===
 
-Automounter for NFS.
+# Compatibility
+
+This module has been tested to work on the following systems with Puppet v3 and Ruby versions 1.8.7, 1.9.3 and 2.0.0.
```

Comment:
With the new travis.yml, this will also support ruby 2.1.0 Puppet v3 with the future parser and Puppet v4